microcosm_pubsub/daemon.py

```diff
@@ -82,7 +82,7 @@ def defaults(self):
         if self.sqs_event:
             config.update(
                 sqs_envelope=dict(
-                    strategy_name=NaiveSQSEnvelope.__name__,
+                    strategy_name=RawSQSEnvelope.__name__,
```

Comment:
According to documentation in `envelope.py` module, `NaiveSQSEnvelope` used for debugging